src/arena/test_util.rs

```diff
@@ -76,5 +90,39 @@ pub fn assert_valid_game_state(game_state: &GameState) {
         if !game_state.player_active.get(idx) && !game_state.player_all_in.get(idx) {
             assert_eq!(0.0, game_state.player_winnings[idx]);
         }
+
+        let mut minimum_player_actions = 0;
+
+        if game_state.ante > 0.0 {
+            minimum_player_actions += 1;
+        }
+
+        if game_state.num_players > 2 { // Full-ring
+            let left_of_dealer_idx = (game_state.dealer_idx + 1) % game_state.num_players;
+
+            if idx == left_of_dealer_idx && game_state.small_blind > 0.0 {
+                minimum_player_actions += 1;
+            }
+
+            let two_left_of_dealer_idx = (left_of_dealer_idx + 1) % game_state.num_players;
+
+            if idx == two_left_of_dealer_idx && game_state.big_blind > 0.0 {
+                minimum_player_actions += 1;
+            }
```

Comment:
This will be moved to assert either that the vector storage or the stats storage of the historian matches the expectation.